lib/gltfStatistics.js

```diff
@@ -0,0 +1,202 @@
+'use strict';
+var Cesium = require('cesium');
+
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+
+module.exports = {
+    gltfStatistics,
+    getDrawCallStatistics,
+    printStats
+}
```

Comment:
Rename `gltfStatistics` to `getStatistics` and `printStats` to `printStatistics`.